src/components/create/BtcInputAmount.vue

```diff
@@ -209,5 +209,12 @@ export default class BtcInputAmount extends Vue {
       this.amountStyle = this.stepState === 'done' ? 'green-box' : 'yellow-box';
     }
   }
+
+  created() {
+    const isInitalValue = this.pegInTxState.amountToTransfer.toBTCString() === '0.00000000';
+    if (!isInitalValue) {
```

Comment:
```suggestion
    const isInitialValue = this.pegInTxState.amountToTransfer === 0;
    if (!isInitialValue) {
```

Reply:
"this.pegInTxState.amountToTransfer" can't be equal to "0" becacuse it's Satoshi type.

Reply:
use `.toNumber()` instead `.toBTCString()`